packages/dev/docs/pages/react-spectrum/styling.mdx

```diff
@@ -0,0 +1,204 @@
+<!-- Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. -->
+
+import sharedDocs from 'docs:@react-types/shared/src/style.d.ts';
+import {PropTable} from '@react-spectrum/docs';
+
+import {Layout} from '@react-spectrum/docs';
+export default Layout;
+
+```tsx import
+import {TextField} from '@react-spectrum/textfield';
+import {ActionButton} from '@react-spectrum/button';
+```
+
+---
+category: Concepts
+---
+
+# Styling
+
+This page describes how styling works in React Spectrum, including how to customize spacing, sizing,
+and positioning, and how to create your own custom components using Spectrum styles.
+
+## Introduction
+
+React Spectrum components are designed to be **consistent** across all Adobe applications. They include builtin
+styling that has been considered carefully, and extensively tested. In general, customizing Spectrum design
+is discouraged, but most components do offer control over layout and other aspects. In addition, you can use 
+Spectrum defined variables to ensure your application conforms to design requirements, and is adaptive across 
+platform scales and color schemes.
+
+## Style props
+
+All React Spectrum components support a limited set of styling options, including layout, spacing, sizing,
+and positioning options. These options only affect things external to the component, not internal styles.
+For example, margins can be set on all components since it is external to the component, but not padding,
+which is internal to the the component (and included in the Spectrum design). Additionally, options like
+colors, borders, text styles, etc. are not available to override.
+
+All of these options are available as props on each React Spectrum component. The following example shows 
+a text field and a button. The text field has a custom width set on it, and the button has a margin before it.
```

Comment:
Should we say that although margin exists, we recommend using Flex or Grid in most cases?

Reply:
this example is a good one because it'll do the wrong thing in RTL

Reply:
heh yeah. so... I'm trying to demonstrate the style props on each item here, so didn't want to bring in `<Flex>`.

Reply:
Right, I don't want to bring it in to the example, but I would like us to point out that this example would not work RTL. Otherwise I have very little faith people will recognize when to use margin like they've always done vs Flex/Grid